bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableScanConfiguration.java

```diff
@@ -273,9 +273,8 @@ public CloudBigtableScanConfiguration build() {
     public ReadRowsRequest get() {
```

Comment:
I think we should deprecate bigtable protos on the public surface (ie Builder#withRequest)

Reply:
I added `@deprecated` annotation for now, and will clean up `CloudBigtableScanConfiguration` and `TemplateUtils` in a separate pr.